src/components/Person/Person.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './Person.scss';
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is {name}
+    </h2>
+    {age &&
+      <p className="Person__age">
+        I am {age}
+      </p>
+    }
+    <p className="Person__partner">
+      {!partnerName && 'I am not married'}
+      {partnerName && sex === 'm'
+        && `My wife's name is ${partnerName}`}
+      {partnerName && sex === 'f'
+        && `My husband's name is ${partnerName}`}
```

Comment:
use ternary operator
if there's no partnerName render 'I am not married', if yes, render their name and check sex inside string with ternary and use wife or husband options